corehq/messaging/scheduling/views.py

```diff
@@ -143,6 +144,9 @@ def get_scheduled_broadcast(self, broadcast_id):
 
     def get_scheduled_broadcast_activate_ajax_response(self, active_flag, broadcast_id):
         broadcast = self.get_scheduled_broadcast(broadcast_id)
+        if not self.can_use_inbound_sms and broadcast.schedule.memoized_uses_sms_survey:
+            return HttpResponseBadRequest()
```

Comment:
Is it worth adding the reason for the failure to the request?

Reply:
we shouldn't actually ever hit this because the ui will restrict it but added the reason in case it ever helps in debugging: https://github.com/dimagi/commcare-hq/pull/19217/commits/343ea2865dd5333ef649efdd0248f6f3af39297f